src/components/Overlay.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Tags from './Tags'
+import { ReactComponent as Arrow } from '../icons/back-arrow.svg'
+import { ReactComponent as Share } from '../icons/share.svg'
+import Contributors from './Contributors'
+import Detail from './Detail'
+
+export default function Overlay({ project, onBack }) {
+  return (
+    <OverlaySection>
+      <Headline>
+        <ArrowStyled onClick={onBack} />
+        <Title>{project.title}</Title>
+        <ShareStyled />
+      </Headline>
+      <Img alt='' src={project.image ?? 'https://source.unsplash.com/random'}></Img>
+      <TagList>
+        {project.tags.map((tag, index) => (
+          <Tags content={tag} key={tag} />
```

Comment:
Tag doesn`t sound very unique to me. Like Martin suggested you go for ID as a key value. (If available)

Reply:
I will use IDs when implement the backend, this needs to be sufficient until then